src/components/shared/formItems/ProjectTicketMods.tsx

```diff
@@ -279,7 +281,10 @@ export default function ProjectTicketMods({
             </span>
           </div>
           <div>
-            <Trans>{100 - total}%to</Trans> <FormattedAddress address={owner} />
+            <Trans>
+              {100 - total}% to{' '}
+              {owner ? <FormattedAddress address={owner} /> : 'project owner'}
```

Comment:
```suggestion
              {owner ? <FormattedAddress address={owner} /> : t`project owner`}
```

Reply:
**question**: I'm wondering how it's possible for `owner` to be undefined here...

Reply:
On create project when a user doesn't have their wallet connected. See screenshots attached

Reply:
oh of course lol. Thanks for explanation 🙏